test/integration/standard_filter_test.rb

```diff
@@ -888,7 +920,7 @@ def test_all_filters_never_raise_non_liquid_exception
       { foo: "bar" },
       [{ "foo" => "bar" }, { "foo" => 123 }, { "foo" => nil }, { "foo" => true }, { "foo" => ["foo", "bar"] }],
       { 1 => "bar" },
-      ["foo", 123, nil, true, false, Drop, ["foo"], { foo: "bar" }],
+      ["foo", 123, nil, true, false, ["foo"], { foo: "bar" }],
```

Comment:
Perhaps the intention was to test against a drop instance here, which should be supported

```suggestion
      ["foo", 123, nil, true, false, Drop.new, ["foo"], { foo: "bar" }],
```